python/ray/util/client/dataclient.py

```diff
@@ -59,9 +68,12 @@ def _next_id(self) -> int:
     def _start_datathread(self) -> threading.Thread:
         return threading.Thread(target=self._data_main, args=(), daemon=True)
 
+    def _start_keepalive_thread(self) -> threading.Thread:
+        return threading.Thread(
+            target=self._keepalive_main, args=(), daemon=True)
```

Comment:
Hmm I wonder if we can have some common `_run_keepalive` function that takes in an event & a stub? That way the dataclient & logclient can share this function!